core/vm/jump_table_export.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package vm
+
+import "strings"
+
+// LookupInstructionSet returns the constructor for an instructionset for the
+// given fork.
+// If the fork is unknown, this method returns nil
+func LookupInstructionSet(fork string) func() JumpTable {
```

Comment:
Just curious why `LookupInstructionSet` instead of `LookupJumpTable`? Doesn't seem to be that we use the "instruction set" terminology much?

Reply:
Myeah... I don't know. The term 'instructionset' seemed more like real-world terminology, as opposed to our "jumptable" which is a bit internal? 
Also, it's not a jumptable, since it's not a table of "computed gotos" to jump to, it's a table of methods.